main.js

```diff
@@ -7,15 +7,22 @@
 //- call display func
 //output: the list display in screen
 const getLatestTutorials = async () => {
-    //fetch API
+    //1/fetch API
+    //The original API is https://www.sesvtutorial.com/page-data/tutorials/page-data.json. Since fetching this API using the Fetch API causes a CORS error, there are two ways to fix this:
```

Comment:
`Since fetching this API using the Fetch API causes a CORS error` a thấy em giải thích nguyên nhân là ntn. nhưng ở dưới vẫn dùng fetch(). thế thì nguyên nhân này chưa chính xác thì phải